submissions/ustymchyk/js-post-oop/index.js

```diff
@@ -1,23 +1,27 @@
 /* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
    Complete the below for code reviewers' convenience:
-
    Code repository: _put repo URL here_
    Web app: _put project's github pages URL here_
    */
 
 // ======== OBJECTS DEFINITIONS ========
 // Define your objects here
 class Creature {
-    constructor(name, gender, age) {
+    constructor(name, gender, age, species) {
         if (new.target === Creature) {
             throw new TypeError("Cannot construct Abstract instances directly");
         }
 
+        this.species = species;
         this.name = name;
         this.gender = gender;
         this.age = age;
     }
 
+    toString() {
+        return `${this.species}, ${this.name}, ${this.gender}, ${this.age}`;
```

Comment:
Imagine, you have dozens of props to form a string from.
How about emploing `Array.join`?

Reply:
Do i understand correctly? Just Array.join([this.species, this.name, this.gender, this.age]).join(', ');

Reply:
Or its about `Object.values(this).join(', ')`? 
But then i`ll have methods code.

Reply:
`[this.species, this.name, this.gender, this.age].join(',  ');` as per changes is totally fine.